### ISSUE TITLE
Точки на карте.

### ISSUE THREAD

**Issue by volerster** [RussianFox/imap#16]
Добрый день.

Подскажите планируется добавление просто точек на карту, для обозначения как проложен кабель? Готов внести небольшую лепту в развите проекта, тестированием.

**Comment by RussianFox** [comment]
Да, планируется поддержка на карте фигур: ломанных, областей и окружностей. С привязкой к триггерам и показаниям элементов данных.
Как только выложу релиз 1.1.6, создам ветку для релиза 2.0.0, оттуда можно будет тестировать.
Создам в этой теме автоматическое оповещение.

**Comment by RussianFox** [comment]
Вполне возможно, что вы не в ту сторону копаете. Если у вас много кабелей и есть схемы сетей, намного проще будет эти схемы оцифровать и использовать в качестве подложки.
Если это так, то могу рассказать, как делать.

**Comment by volerster** [comment]
Как нарисовать я их знаю, а как подложить напишите если не сложно. Еще подписи можно у вас добавлять? 

24.05.2015, 21:42, "RussianFox" <notifications@github.com>:
> Вполне возможно, что вы не в ту сторону копаете. Если у вас много кабелей и есть схемы сетей, намного проще будет эти схемы оцифровать и использовать в качестве подложки.
> Если это так, то могу рассказать, как делать.
>
> —
> Reply to this email directly or view it on GitHub.

-- 
С уважением, Валерий.
+79265504470

**Comment by RussianFox** [comment]
Смысл в том, чтобы выяснить координаты левого верхнего и нижнего правого углов картинки. Тут нужно какое-то ПО. Вот пример: http://npk-kaluga.ru/GeoRefMI_2.htm

Далее, смотрите пример в файле imap/additions.js.template

**Comment by volerster** [comment]
Хорошо, но жду и вашего варианта. И для забикс 2.4.5 ваша версия пойдет?

**Comment by RussianFox** [comment]
2.4.5 самая часто используемая версия с imap

**Comment by montino** [comment]
Спасибо за работу
